lib/jsdom/browser/Window.js

```diff
@@ -36,14 +33,14 @@ const WindowEventHandlersImpl = require("../living/nodes/WindowEventHandlers-imp
 // NB: the require() must be after assigning `module.exports` because this require() is circular
 // TODO: this above note might not even be true anymore... figure out the cycle and document it, or clean up.
 module.exports = Window;
-const dom = require("../living");
+const bootstrapDOM = require("../living/generated/bundle-entry").bootstrap;
+const extraDOM = require("../living");
```

Comment:
I'd like us to carefully consider our names here. E.g. instead of "extraDOM", maybe "nonGeneratedInterfaces" or similar.